packages/upstream/ocb-stubblr.0.1.1/opam

```diff
@@ -15,7 +15,3 @@ depends: [
   "astring" ]
 depopts: []
 build: [ "ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%" "--tests" "false" ]
-patches: [
-    "0001-bytecode-custom-needs-cclib-as-well.patch"
```

Comment:
It does not seem that you are removing the parch file. Double check if we put it there before removing it

Reply:
The patch file was introduced by you and as far as I can tell the change it introduces is not current version. So we better keep it.

https://github.com/pqwy/ocb-stubblr/pull/11